src/directory/directory.mjs

```diff
@@ -820,6 +820,46 @@ export const directory = {
           }
         ]
       },
+      logging: {
+        title: 'Logging',
+        items: [
+          {
+            title: 'Setup Logging',
```

Comment:
Can we please change the title to "Set up Logging"

Reply:
Title has been updated.